src/i_joystick.c

```diff
@@ -78,6 +73,13 @@ static int joystick_strafe_invert = 0;
 static int joystick_look_axis = -1;
 static int joystick_look_invert = 0;
 
+// Configurable dead zone for each axis, specified as a percentage of the axis
+// max value.
+static int joystick_x_dead_zone = 33;
```

Comment:
These defaults sound rather large. I think generally 5-10% is recommended.

Reply:
The previous hardcoded deadzone was `(32768 / 3)` which is one third of the range output by `SDL_JoystickGetAxis` and `SDL_GameControllerGetAxis`. I trust that this value was selected for a good reason, so I kept it as the default here.